### ISSUE TITLE
redis.Cluster   [ioredis] Unhandled error event: Error: Connection is closed.

### ISSUE THREAD

**Issue by frank198** [luin/ioredis#337]
ioredis version:2.0.1
I use redis cluster connect
```sh
let redisClusterConfig=[
            {
                host:"127.0.0.1",
                port:7001
            },
            {
                host:"127.0.0.1",
                port:7002
            },
            {
                host:"127.0.0.1",
                port:7003
            },
            {
                host:"127.0.0.1",
                port:7004
            },
            {
                host:"127.0.0.1",
                port:7005
            },
            {
                host:"127.0.0.1",
                port:7006
            }
        ]
let redisClient = new redis.Cluster(redisCluster);
```

Error：
```sh
[ioredis] Unhandled error event: Error: Connection is closed.
    at close (/home/frank/Project/NodeJs/ccjh/node_modules/ioredis/lib/redis/event_handler.js:99:21)
    at Socket.<anonymous> (/home/frank/Project/NodeJs/ccjh/node_modules/ioredis/lib/redis/event_handler.js:74:14)
    at Socket.g (events.js:286:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at TCP._handle.close [as _onclose] (net.js:492:12)
```

**Comment by luin** [comment]
Seems to be fixed in https://github.com/luin/ioredis/pull/322. Not released though.

**Comment by luin** [comment]
Should be fixed in v2.1.0. Closing...
